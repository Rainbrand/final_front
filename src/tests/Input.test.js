import React from "react";
import {shallow} from "enzyme";
import Input from "../Components/Input";

describe("Input component should:", () => {
    let component

    beforeAll(() => {
        component = shallow(<Input ref="ref" className="chat__input"/>)
        console.log(component.debug())
    })

    test("render Input component", () => {
        expect(component).toMatchSnapshot()
    })

    test("have props", () => {
        expect(component.props()).not.toBeUndefined()
    })

    describe("get correct props:", () => {
        test("className", () => {
            expect(component.props().className).toEqual("chat__input")
        });
    })
})
