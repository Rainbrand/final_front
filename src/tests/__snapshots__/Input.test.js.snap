// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should: render Input component 1`] = `ShallowWrapper {}`;
